Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -17,10 +17,10 @@ import Foundation
 import PackageModel
 import SourceControl
 
-import class TSCBasic.BufferedOutputByteStream
-import struct TSCBasic.ByteString
 import class Basics.AsyncProcess
 import struct Basics.AsyncProcessResult
+import class TSCBasic.BufferedOutputByteStream
+import struct TSCBasic.ByteString
```

Comment:
It's unclear at a glance which parts of the diff actually resolve the issue. We generally prefer to keep formatting changes to separate PRs, especially those affecting the whole file.